src/base/static/js/views/app-view.js

```diff
@@ -696,23 +700,31 @@ module.exports = Backbone.View.extend({
     var self = this;
 
     if (!this.placeFormView) {
-      this.placeFormView = new PlaceFormView({
-        appView: this,
-        router: this.options.router,
-        placeConfig: this.options.placeConfig,
-        mapConfig: this.options.mapConfig,
-        userToken: this.options.userToken,
-        geometryEditorView: this.mapView.geometryEditorView,
-        collectionsSet: {
-          places: this.places,
-          landmarks: this.landmarks,
-        },
-      });
+      // this.placeFormView = new PlaceFormView({
+      //   appView: this,
+      //   router: this.options.router,
+      //   placeConfig: this.options.placeConfig,
+      //   mapConfig: this.options.mapConfig,
+      //   userToken: this.options.userToken,
+      //   geometryEditorView: this.mapView.geometryEditorView,
+      //   collectionsSet: {
+      //     places: this.places,
+      //     landmarks: this.landmarks,
+      //   },
+      // });
     }
 
+    ReactDOM.render(
+      <InputForm placeConfig={this.options.placeConfig} 
+                 mapConfig={this.options.mapConfig} />,
+      document.querySelector("#content article")
```

Comment:
Probably a way to scope this to the backbone component instead of a global selector.